test/e2e/multihoming.go

```diff
@@ -803,6 +807,38 @@ var _ = Describe("Multi Homing", func() {
 						Expect(err).NotTo(HaveOccurred())
 						Expect(inRange(newDesiredRange, clientIP)).To(Succeed())
 					})
+
+					It("can no longer communicate over a localnet secondary network from pod to the underlay service", func() {
+						clientPodConfig := podConfiguration{
+							name:        clientPodName,
+							namespace:   f.Namespace.Name,
+							attachments: []nadapi.NetworkSelectionElement{{Name: secondaryNetworkName}},
+						}
+						kickstartPod(cs, clientPodConfig)
+
+						By("asserting the *client* pod can no longer contact the underlay service")
+						Eventually(connectToServer(clientPodConfig, underlayServiceIP, servicePort)).ShouldNot(Succeed())
```

Comment:
This eventually should receive a closure or do as 
```golang
Eventually(connectToServer).WithArguments(clientPodConfig, underlayServiceIP, servicePort).ShouldNot(Succeed())
```

Reply:
DONE on both occurrences.